gloo/docs/installation/enterprise/_index.md

```diff
@@ -120,6 +120,7 @@ The table for gloo open-source overrides (also available in enterprise) is [here
 | grafana.enabled                                           | bool     | deploy grafana in your gloo system namespace. default is `true` |
 | prometheus.enabled                                        | bool     | deploy prometheus in your gloo system namespace. default is `true` |
 | rateLimit.enabled                                         | bool     | deploy rate-limiting in your gloo system namespace. default is `true` |
+| global.extensions.extAuth.envoySidecar                    | bool     | deploy ext-auth in the gateway-proxy pod, as a sidecar to envoy. communicates over unix domain socket instead of TCP. default is `false` |
```

Comment:
```suggestion
| global.extensions.extAuth.envoySidecar                    | bool     | Deploy ext-auth in the gateway-proxy pod, as a sidecar to envoy. Communicates over unix domain socket instead of TCP. Default is `false` |
```

Reply:
the rest of the table is lower-cased, which also matches most of the entries here: https://gloo.solo.io/installation/gateway/kubernetes/#list-of-gloo-helm-chart-values

i think we should leave it lowercase for now, and if we update that, make the change everywhere.